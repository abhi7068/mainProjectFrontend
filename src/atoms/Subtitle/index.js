import React from 'react';

import { makeStyles } from '@material-ui/core/styles';
import PropTypes from 'prop-types';
const useStyles = makeStyles((theme) => ({
  subtitle: {
    fontSize: '18px',
    marginTop: '16px',
    lineHeight: '150%',
  },
}));

function Subtitle({ children }) {
  const classes = useStyles();
  return <p className={classes.subtitle}>{children}</p>;
}
Subtitle.propTypes = {
  children: PropTypes.oneOfType([PropTypes.string, PropTypes.func]),
};
export default Subtitle;
