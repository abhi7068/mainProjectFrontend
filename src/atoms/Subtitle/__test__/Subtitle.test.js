import React from 'react';

import Subtitle from '../index';
import { render } from '@testing-library/react';

it('it renders', () => {
  render(<Subtitle />);
});
it('should take a snapshot', () => {
  const { asFragment } = render(<Subtitle />);
  expect(asFragment(render(<Subtitle />))).toMatchSnapshot();
});
