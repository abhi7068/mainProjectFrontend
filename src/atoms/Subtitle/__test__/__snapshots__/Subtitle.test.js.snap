// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a snapshot 1`] = `
<DocumentFragment>
  <p
    class="makeStyles-subtitle-2"
  />
</DocumentFragment>
`;
